packages/cli/configuration/fern/definition/docs.yml

```diff
@@ -449,6 +451,14 @@ types:
       - fixed
       - static
 
+  AudienceId: string
+
+  Audience:
+    discriminated: false
+    union:
+      - AudienceId
+      - list<AudienceId>
```

Comment:
do you think there should be a top level place in docs.yml where a user should declare their audiences? in the fern definition they have to list them all out in `api.yml`

Reply:
then you can have a `fern check` rule if an audience goes unused etc, instead of trying to compute all the different sets

Reply:
Agreed!

Reply:
rule added!